keystone/files/mitaka/keystone.conf.Debian

```diff
@@ -2193,3 +2193,52 @@ driver = memcache_pool
 
 [extra_headers]
 Distribution = Ubuntu
+
+{% if server.get('k2k') %}
```

Comment:
Check ldap part with include _ldap.conf, you want to do something very similar.
This won't work if k2k is not defined etc.:
```{%- if server.k2k is defined %}```